lib/vehicle.rb

```diff
@@ -0,0 +1,32 @@
+class Vehicle
+  attr_reader :year, :make, :model, :vehicle, :passengers, :num_adults
```

Comment:
```suggestion
  attr_reader :year, :make, :model, :vehicle, :passengers
```

You have defined `num_adults` below on line 23. Remember an `attr_reader` creates a method for us so by doing this you have two methods by the same name which can create conflict. We only need the method on line 23.